external/leveldb/leveldb_tuning.ipynb

```diff
@@ -0,0 +1,767 @@
+      "  \"homogeneous_random_forest_regression_model_config.n_estimators\": 10,\n",
+      "  \"homogeneous_random_forest_regression_model_config.features_fraction_per_estimator\": 1,\n",
+      "  \"homogeneous_random_forest_regression_model_config.samples_fraction_per_estimator\": 0.9,\n",
+      "  \"homogeneous_random_forest_regression_model_config.regressor_implementation\": \"DecisionTreeRegressionModel\",\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.criterion\": \"mse\",\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.splitter\": \"best\",\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.max_depth\": 0,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_samples_split\": 2,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_samples_leaf\": 3,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_weight_fraction_leaf\": 0,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.max_features\": \"auto\",\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.max_leaf_nodes\": 0,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_impurity_decrease\": 0,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.ccp_alpha\": 0,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_samples_to_fit\": 10,\n",
+      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.n_new_samples_before_refit\": 1,\n",
+      "  \"homogeneous_random_forest_regression_model_config.bootstrap\": 1,\n",
+      "  \"experiment_designer_config.utility_function_implementation\": \"ConfidenceBoundUtilityFunction\",\n",
+      "  \"experiment_designer_config.numeric_optimizer_implementation\": \"RandomSearchOptimizer\",\n",
+      "  \"experiment_designer_config.confidence_bound_utility_function_config.utility_function_name\": \"upper_confidence_bound_on_improvement\",\n",
+      "  \"experiment_designer_config.confidence_bound_utility_function_config.alpha\": 0.1,\n",
+      "  \"experiment_designer_config.random_search_optimizer_config.num_samples_per_iteration\": 1000,\n",
+      "  \"experiment_designer_config.fraction_random_suggestions\": 0.5,\n",
+      "  \"experiment_designer_config_fraction_random_suggestions\": 0.1\n",
+      "}.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Optimization Problem\n",
+    "# You might have to change the min and max value based on the objective that you are using \n",
```

Comment:
@nithinv13 this note probably needs more details.
Actually, a better way would be to give the `ContinuousDimension` parameter some better names than `objective` so that it's more clear that if they're tuning for "latency" they want ranges from `(0, n]` some time units based on their hardware vs. if they're tuning for "throughput" they want ranges from `(0, m]` MB/s based on their hardware.

Either way, I think providing slightly higher bounds than are expected is good so that you don't accidentally end up capping the optimization problem's output if your benchmarks somehow exceed that.